ricecooker/chefs.py

```diff
@@ -281,10 +282,49 @@ def save_channel_tree_as_json(self, channel):
         self.CHEF_RUN_DATA['tree_archives']['current'] = filename.replace(os.getcwd() + '/', '')
         self.save_chef_data()
 
+    def save_channel_metadata_as_csv(self, channel):
+        # create data folder in chefdata
+        # config.LOGGER.info('SAVE METADATA TO CSV')
+        DATA_DIR = os.path.join('chefdata', 'data')
+        if not os.path.isdir(DATA_DIR):
+            os.makedirs(DATA_DIR)
+        metadata_csv = csv.writer(open(os.path.join(DATA_DIR, 'content_metadata.csv'), 'w', newline=''))
+        headers = [
+            'Source_id',
+            'Topic Structure',
+            'Old Title',
+            'New Title',
+            'Old Description',
+            'New Description',
+            'Old Tags',
+            'New Tags',
+            'Last Modified'
+        ]
+        metadata_csv.writerow(headers)
+
+        channel.save_channel_children_to_csv(metadata_csv)
+
 
     def save_chef_data(self):
         json.dump(self.CHEF_RUN_DATA, open(config.DATA_PATH, 'w'), indent=2)
 
+    def save_modifications(self, contentNode, metadata_dict = {},metadata_csv = None):
```

Comment:
Could we call this `apply_modifications` instead?

Reply:
I'm still seeing this as `save_modifications` in the latest version.

Reply:
Also, this function doesn't appear to use `metadata_csv` at all, so we can remove it.

Reply:
The above issues all still appear in the latest version of the code.

Reply:
Sorry about that. Changed function name to apply_modifications